api/src/main/java/org/openmrs/scheduler/SchedulerService.java

```diff
@@ -103,6 +103,16 @@
 	@Authorized( { "Manage Scheduler" })
 	public TaskDefinition getTask(Integer id);
 	
+	/**
+	 * @since 2.4.0
+	 * 
+	 *        Get task by given UUID.
```

Comment:
Can you look at the formatting. Also, take another look at the grammar used by the existing openmrs api javadocs.

Reply:
fixed it,hope its clean now